hicommonwealth/commonwealth
# Contest/Farcaster contest fixes

## Link to Issue
Closes: #10083
Closes: #10084
Closes: #10086
Closes: #10087

## Description of Changes
- copy changes
- removed check eligibility button from frame
- adjusted path to cast replies to be able to display empty placeholder



## Test Plan
- check copy changes in #10083 and #10084
- make sure there is no check eligibility button in frame
- make sure there is a placeholder in leaderboard page when there is no entries yet

## Deployment Plan

1. n/a

## Other Considerations

- n/a
